docs/custom-workload.md

```diff
@@ -0,0 +1,97 @@
+# Deploying a custom workload
+
+## Example workload
+
+An example windows deployment would look like this.
```

Comment:
capitalize Windows please, there's a couple more instances of this later in the file